compiler/src/utils/markdown.re

```diff
@@ -15,26 +15,60 @@ let code = code => {
   Format.sprintf("`%s`", code);
 };
 
-let table = (~headers: list(string), rows) => {
+let table = (~nested=false, ~headers: list(string), rows) => {
```

Comment:
I don't like boolean flags like this, we should think through a real API that is needed if we are putting tables inside